public/stylesheets/sass/pages/_alnylam.scss

```diff
@@ -0,0 +1,48 @@
+#alnylam{
+  background-color: #0099cc;
+
+  .char {
+    animation-delay: var(--del);
+    animation-direction: var(--dir, normal);
+    animation-duration: var(--dur);
+    animation-iteration-count: infinite;
+    animation-name: var(--name);
+    animation-timing-function: var(--tf);
+    display: inline-block;
+    position: relative;
+    transform-origin: 50% 100%;
+    z-index: 1;
+  }
+
+  .twirl {
```

Comment:
holy shit

Reply:
there exists a js library that auto splits words into individual characters in their own span with '.char' class. v useful

https://splitting.js.org/
https://github.com/laudickson/portfolio/pull/19/files#diff-520e8d912e4640d2a35edc95284d9e7e35c31a5ed6684872ccd4f946184f2946R10